build_debian.sh

```diff
@@ -571,6 +572,9 @@ sudo du -hsx $FILESYSTEM_ROOT
 sudo mkdir -p $FILESYSTEM_ROOT/var/lib/docker
 sudo mksquashfs $FILESYSTEM_ROOT $FILESYSTEM_SQUASHFS -e boot -e var/lib/docker -e $PLATFORM_DIR
 
+scripts/collect_host_image_version_files.sh $TARGET_PATH $FILESYSTEM_ROOT
+sudo LANG=C chroot $FILESYSTEM_ROOT set_build_hooks -d
```

Comment:
This line is already covered by above line.

Reply:
Removed.

---
In reply to: [539813194](https://github.com/Azure/sonic-buildimage/pull/5718#discussion_r539813194) [](ancestors = 539813194)